src/System.Private.ServiceModel/tests/Scenarios/Security/TransportSecurity/Https/HttpsTests.cs

```diff
@@ -111,13 +124,91 @@ public static void Https_SecModeTrans_ClientCredTypeNone_ServerCertValModePeerOr
         string testString = "Hello";
         ChannelFactory<IWcfService> factory = null;
         IWcfService serviceProxy = null;
+        CommunicationException communicationException = null;
 
         try
         {
             // *** SETUP *** \\
             BasicHttpsBinding binding = new BasicHttpsBinding(BasicHttpsSecurityMode.Transport);
             binding.Security.Transport.ClientCredentialType = HttpClientCredentialType.None;
-            endpointAddress = new EndpointAddress(new Uri(Endpoints.Https_SecModeTrans_ClientCredTypeNone_ServerCertValModePeerTrust_Address));
+            endpointAddress = new EndpointAddress(new Uri(
+                                Endpoints.Https_SecModeTrans_ClientCredTypeNone_ServerCertValModePeerOrChainTrust_Address));
```

Comment:
Is this endpoint correct? 
The test is named Https_SecModeTrans_CertValMode_PeerTrust_Fails_Not_In_TrustedPeople
but you're using a cert that is capable of doing PeerOrChainTrust?